purchase_order_downpayment/wizard/__init__.py

```diff
@@ -0,0 +1 @@
+from . import purchase_order_downpayment_wiz
```

Comment:
```suggestion
from . import purchase_order_downpayment_wizard
```

ususally in OCA, use the name of the model as the name of the file.